src/integration.test.js

```diff
@@ -0,0 +1,134 @@
+const { ApolloServer } = require("apollo-server-cloud-functions");
+
+const typeDefs = require("./schema");
+const { getResolvers } = require("./resolvers");
+const {
+  getOrderedCatalogMock,
+  getLatestListMock,
+  addItemToCatalogMock,
+  getDocumentMock,
+} = require("./service_mocks");
+
+let server = null;
+
+beforeAll(() => {
+  const resolvers = getResolvers(
+    getOrderedCatalogMock,
+    getLatestListMock,
+    addItemToCatalogMock,
+    getDocumentMock
+  ).then((resolvers) => {
+    server = new ApolloServer({
+      typeDefs: typeDefs,
+      resolvers: resolvers,
+    });
+  });
+
+  return resolvers;
+});
+
+it("fetches category list", async () => {
+  // create a test server to test against, using our production typeDefs and resolvers
+  const res = await server.executeOperation({
+    query:
+      "query TestQuery {\
+              catalog {name, category, defaultUnit}\
+            }",
```

Comment:
Is it possible to use template tags instead?

```suggestion
      "query TestQuery {\
              catalog {name, category, defaultUnit}\
            }",
```